examples/logging-example/README.md

```diff
@@ -0,0 +1,58 @@
+# Logging Example
+
+In order to run this example, you need to be running a _Tenant Security Proxy_ (TSP) on your machine.
```

Comment:
I know this is what we have other places, but we actually want TSP + Logdriver, which is done via the docker compose.